kotsadm/web/src/components/UploadAirgapBundle.jsx

```diff
@@ -294,11 +297,34 @@ class UploadAirgapBundle extends React.Component {
     }, 1000);
   }
 
+  getSupportBundleCommand = async (slug) => {
+    const res = await fetch(`${window.env.API_ENDPOINT}/troubleshoot/app/${slug}/supportbundlecommand`, {
+      method: "GET",
+      headers: {
+        "Authorization": Utilities.getToken(),
+      }
+    });
+    if (!res.ok) {
+      throw new Error(`Unexpected status code: ${res.status}`);
```

Comment:
this should also setState to show error in modal.

Reply:
it does